modules/cnf-installing-the-operators.adoc

```diff
@@ -0,0 +1,273 @@
+// CNF-950 4.7 Installing the operators
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-installing-the-operators_{context}"]
+= Installing the operators
+
+[id="cnf-installing-the-performnce-addon-operator_{context}"]
+== Installing the Performance Addon Operator
+
+Install the Performance Addon Operator using the {product-title} CLI or the web console.
+
+.Procedure
+
+. Create the `operator-namespace.yaml` file:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    openshift.io/cluster-monitoring: "true"
+  name: openshift-performance-addon-operator
+spec: {}
+----
+
+. Create the `operator_catalogsource.yaml` file:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: performance-addon-operator
+  namespace: openshift-marketplace
+spec:
+  displayName: Openshift Performance Addon Operator
+  icon:
+    base64data: ""
+    mediatype: ""
+  image: quay.io/openshift-kni/performance-addon-operator-index:4.7-snapshot
+  publisher: Red Hat
+  sourceType: grpc
+----
+
+. Create the `operator_operatorgroup.yaml` file:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: performance-addon-operator
+  namespace: openshift-performance-addon-operator
+----
+
+. Create the `operator_subscription.yaml` file:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: performance-addon-operator
+  namespace: openshift-performance-addon-operator
+spec:
+  channel: "4.7"
```

Comment:
@StephenJamesSmith If I look at the PAO official documentation, there is a procedure for obtaining the default channel and assigning it. Could it be possible to adopt this procedure here?
```suggestion
  Run the following command to get the channel value required for the next step.

$ oc get packagemanifest performance-addon-operator -n openshift-marketplace -o jsonpath='{.status.defaultChannel}'

Example output

4.6

Create the following Subscription CR and save the YAML in the pao-sub.yaml file:
Example Subscription

apiVersion: operators.coreos.com/v1alpha1
kind: Subscription
metadata:
  name: openshift-performance-addon-operator-subscription
  namespace: openshift-performance-addon-operator
spec:
  channel: "<channel>" 
  name: performance-addon-operator
  source: redhat-operators 
  sourceNamespace: openshift-marketplace

	Specify the value from you obtained in the previous step for the .status.defaultChannel parameter.
```

Reply:
@vitus133 Line 69 is within a code block. Do we want to remove only line 69? Or do we want to replace it with channel: "${OC_VERSION}"?

Also, your suggested changes looks 3 additional steps?

Reply:
Sorry, what I meant is replacing this step and associated code block:
![image](https://user-images.githubusercontent.com/7355270/106893375-25f91180-66f6-11eb-809e-8953d4083aca.png)
by these two steps:
![image](https://user-images.githubusercontent.com/7355270/106893658-8be59900-66f6-11eb-8431-6c60f5207d3a.png)

Reply:
This is to be consistent with other places in the documentation

Reply:
Changes made.